marketplaceItems/audioFocus/appResources/appData/resources/js/audioFocus_ui.js

```diff
@@ -0,0 +1,118 @@
+/*
+    Solo Point
+    Created by Milad Nazeri on 2019-01-07
+    Copyright 2019 High Fidelity, Inc.
+
+    Distributed under the Apache License, Version 2.0.
+    See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+    Point to solo someone to hear them better in a crowd!
+    TABLET JS
+    
+*/
+
+
+// *************************************
+// START UTILITY FUNCTIONS
+// *************************************
+// #region Utilty
+
+var debug = true;
+var FALSE = "false";
+
+// Easy log function checking for message, an object to stringify, and whether it should be enabled or not
+function log(message, object, enabled){
+    if (!debug || enabled === FALSE) {
+        return;
+    }
+
+    var finalMessage;
+
+    finalMessage = "\n\n\t" + message + ":" + "\n\n";
+
+    if (object) {
+        finalMessage += "\n\t\t" + JSON.stringify(object, null, 4) + "\n";
+    }
+
+    print(finalMessage);
+}
+
+// #endregion
+// *************************************
+// END UTILITY FUNCTIONS
+// *************************************
+
+var UPDATE_SOLO = "UPDATE_SOLO";
+var CLEAR_LIST = "CLEAR_LIST";
+
+var avatarList = document.getElementById("avatar-list");
```

Comment:
please move these closer to where they're first used (or where they're first relevant)